python-sdk/src/langwatch/exporters/async_batch_exporter.py

```
@@ -0,0 +1,135 @@
+
+from langwatch.domain import SpanExporterExcludeRule
+
+
+class AsyncBatchExporter(SpanExporter):
+    def __init__(
+        self,
+        exporter: SpanExporter,
+        max_queue_size: int = 1000,
+        max_export_batch_size: int = 100,
+        export_interval: float = 5.0,
+        span_exporter_exclude_rules: List[SpanExporterExcludeRule] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)